lib/CStack.cpp

```diff
@@ -304,6 +304,7 @@ std::vector<BattleHex> CStack::meleeAttackHexes(const battle::Unit * attacker, c
 			res.push_back(otherDefenderPos);
 		}
 	}
+	UNUSED(mask);

```

Comment:
analyser says that `mask` is unused after the last assignment (a few lines above). In case a new condition is added, it'd be very easy to forget to add the assignment back. That's why I mark the variable as unused explicitly.